src/styles/theme.css

```diff
@@ -1,8 +1,29 @@
-body {
-  color: green;
+@font-face {
+  font-family: "Geist-Sans";
+  src: url('../../node_modules/geist/dist/fonts/geist-sans/Geist-Regular.woff2') format("truetype");
 }
 
-h1, h2, h3, h4, h5, h6 {
-  text-align: center;
-  color: black;
+@font-face {
+  font-family: "Geist-Mono";
+  src: url('../../node_modules/geist/dist/fonts/geist-mono/GeistMono-Regular.woff2') format("truetype");
+}
+
+:root {
+  --color-primary: #016341;
+  --color-secondary: #0E0F0C;
+  --color-tertiary: #151818;
+  --color-accent: #C3DDBA;
+  --color-white: #FFFFFF;
+  --color-black: #020202;
+  --color-gray: #F5F5F5;
```

Comment:
@Auhseh 
What is the distinction between the `--color-gray` and `--color-inner-gray`.  I'm guessing these are intended for the docs pages?

Reply:
@Auhseh 
Any thoughts here?  I'm wondering if we just need better names for these, or are these just one-offs to a particular page?